### PR TITLE
Tests: Model, Clear Command, Restore Command

### PR DESCRIPTION
**Pull request by ChuaPingChan**

### CODE REVIEW

File: src/test/java/seedu/multitasky/testutil/CommandUtil.java [hunk 1]
```diff
@@ -173,4 +187,76 @@ public static String getDateDetailsAsInputString(Calendar calendar) {
...
+        return ListCommand.MESSAGE_ACTIVE_SUCCESS;
+    }
+    // @@ author
+
+    // @@author A0126623L
+    /**
+     * Returns an list command string for listing archive.
+     */
+    public static String getListArchiveCommand() {
+        return ListCommand.MESSAGE_ACTIVE_SUCCESS + " " + CliSyntax.PREFIX_ARCHIVE;
```
Comment (mattheuslee) [inline src/test/java/seedu/multitasky/testutil/CommandUtil.java]: curious why this is combining the success message with the prefix? if it's to get the command should it be `ListCommand.COMMAND_WORD + " " + CliSyntax.PREFIX_ARCHIVE`?

Comment (ChuaPingChan) [reply src/test/java/seedu/multitasky/testutil/CommandUtil.java]: Ah, mistake. I intended to type ListCommand.COMMAND_WORD. Will make the changes when I'm home later

**Comment by ChuaPingChan** [comment]
@mattheuslee 
I'll be out for a while till late evening, more commits will come at 11pm-3am tonight haha. Do you think it'll be helpful to merge the branch first so that others can use the updated SampleEntries? I'll request for approval if so.

**Comment by mattheuslee** [comment]
@ChuaPingChan sure I think we can merge it first

**Review by mattheuslee — APPROVED** [review]

**Comment by ChuaPingChan** [comment]
@CS2103JUN2017-T2/developers 
Hi all, work is still in progress on my part but I'd like to make an early merge first so that `SampleEntries.java` can be used by others.

**Pull request merged**